src/components/TabBarIcon.js

```diff
@@ -1,15 +1,21 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import { Ionicons } from '@expo/vector-icons';
 
 import { colors, normalize } from '../config';
 
+//TODO refactor TabBarIcon to be a function component that can use OrientationContext
```

Comment:
what type of component is it now?

Reply:
it's a default functional component, but we need a "fancy" functional component to be able to use hooks
  which we will need to setup `useContext` for orientation stuff ( from your commits in https://github.com/ikuseiGmbH/smart-village-app-app/pull/94/commits )

Reply:
but it is already a functional component (definition is `function` and not `class`). so maybe it needs to add orientation functionality, but no refactoring to functional component.

Reply:
the TODO comment can now be removed, as you already refactored